### ISSUE TITLE
eslint support?

### ISSUE THREAD

**Issue by namirsab** [Cretezy/Noderize#8]
Would be nice to have eslint configured by default

**Comment by Cretezy** [comment]
Yep, this is on the roadmap! Should be coming in about a week at most, in 0.5.0.
Do you have an suggestions of implementations for ESLint?

**Comment by namirsab** [comment]
Hey, thanks!
As the project is already using prettier, we have to set it up properly so eslint and prettier don't conflict.
It would be nice to have different options to choose which code style you wanna use (airbnb, Google, standard, custom)

**Comment by Cretezy** [comment]
Definitely. I think asking on project creation for a code style is a good idea, or with a flag.

**Comment by chanlito** [comment]
Just wanna say thanks to @Cretezy & all contributors, this project really helped me a lot.

**Comment by hobochild** [comment]
I think following create-reacts style, where you supply a minimal set of rules by default that don't enforce any style just catch common mistakes, but then allow the users to `extend` the noderize rules by creating a .eslintrc in the root of the folder.

**Comment by Cretezy** [comment]
For now, Noderize will ship with the new `lint` script, which links to Eslint. People can then do `yarn lint --init` and set it up however they want.

**Comment by Cretezy** [comment]
https://github.com/Cretezy/Noderize/commit/8644d3818d78a1637f103100b4e29b0f3caa3c7b

Released v0.5.0 with ESLint support.

Add the following in your `package.json`'s `scripts`:

`"lint": "noderize-scripts lint"`

Then use:
```bash
yarn lint --init
# or
npm run lint --init
```

Then:
```bash
yarn lint
# or
npm run lint
```
